app/styles/app.scss

```diff
@@ -556,12 +558,13 @@ hr {
             opacity: 1;
         }
     }
-    .row {
-        margin: 0;
-    }
-    label, input, textarea {
-        display: block;
-    }
+    // THESE STYLES ARE INTERFERING WITH MY STYLES. 
```

Comment:
@pattisdr you can keep these commented out, it would be very hard for you to fix it. I will go through and fix styles, we shouldn't be using such general styles across the app. Let's merge this in first.